### DISCUSSION TITLE
How to Edit OCPP Configuration Keys for Offline Charging?

### DISCUSSION THREAD

**Discussion by MateoGreil** [lbbrhzn/ocpp#1256] [Q&A] [answered]
Hello,

Is there a way to edit the OCPP configuration keys of my charger?
I want to enable offline charging so that the charger continues to work even if my HA is offline.
It looks like this functionality is controlled by an OCPP configuration key (`allow_offline_tx_for_unknown_id`), but I can't find any way to edit these values.

Thanks!

**Comment by drc38** [comment, marked as answer]
You can use the Configure service to change key values.

**Reply by MateoGreil** [reply]
Thanks!
One last question, this is more about HA than this integration, but you've probably the answer :
How to get the response of the service called ?
I want to get the response of the get configuration.

**Reply by drc38** [reply]
It appears in the log

**Reply by MateoGreil** [reply]
ok thanks
